classes/dci.adoc

```diff
@@ -78,6 +78,9 @@ Template id if DCI came from Template or 0.
 `templateItemId => Integer`::
 DCI item id in template if DCI came from Template or 0.
 
+`transformedDataType => String`::
```

Comment:
Integer is returned
Also should have reference to https://netxms.org/documentation/nxsl-latest/#const-dci-datatype